test/mlir/tosa/onnx_lowering.mlir

```diff
@@ -0,0 +1,11 @@
+// RUN: onnx-mlir-opt -O3 --shape-inference --convert-onnx-to-tosa %s -split-input-file | FileCheck %s
+
+func private @test_relu(%arg0 : tensor<?x10xf32>) -> tensor<*xf32> {
+  %0 = "onnx.Relu"(%arg0) : (tensor<?x10xf32>) -> tensor<*xf32>
+  "std.return"(%0) : (tensor<*xf32>) -> ()
```

Comment:
`std` was replaced by `func` in the new LLVM commit. Please use `func.return` instead.

Reply:
Done! Ran cmake --build . --target check-onnx-lit as well: 

Testing Time: 1.33s
  Unsupported: 71
  Passed     : 68
$